service/history/transferQueueProcessorBase.go

```diff
@@ -142,8 +142,9 @@ func (t *transferQueueProcessorBase) pushDecision(task *persistence.TransferTask
 }
 
 func (t *transferQueueProcessorBase) recordWorkflowStarted(
```

Comment:
Could we just pass the serialized memo or at least its thrift struct? I don't see a reason to make it strongly typed here as it is never looked at.

Reply:
make sense. Updated to use binary